### DISCUSSION TITLE
pnpm outdated throws segmentation fault

### DISCUSSION THREAD

**Discussion by cascading-jox** [pnpm/pnpm#5188] [Q&A]
pnpm 7.9.0 throws segmentation fault when running the `outdated` command on rockylinux 8. I have a feeling something went wrong when I either updated nodejs or npm because pnpm complained about the path of something I think and after that the segmentation fault started. Does anyone have any pointers to which logs to check or what to do to repair pnpm?

**Comment by zkochan** [comment]
You can probably try to install the non-bundled version of pnpm. Either using `npm i -g pnpm` or using corepack.

**Reply by cascading-jox** [reply]
I should've mentioned I'm using that version already, installed via npm. If I try to uninstall pnpm using `npm remove pnpm` it fails and says cannot read properties of null (reading 'matches'). Can I force an uninstallation somehow? Or do you think this is related?
